### PR TITLE
feat: added toolbar in multimeter 

### LINKED ISSUE #1293: Add action bar in multimeter
**Actual Behaviour**

Currently there is no action-bar in multimeter

**Expected Behaviour**

There should be an action-bar in multimeter.

**Steps to reproduce it**

Go to multimeter activity

**LogCat for the issue**

Not applicable.

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #1293 

**Changes**: [Add here what changes were made in this issue and if possible provide links.]
Added toolbar in multimeter to further have data recording feature implemented in multimeter.

**Screenshot/s for the changes**: [Add screenshot/s of the layout where you made changes or a `*.gif` containing a demonstration]
 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]
[multimeter_toolbar.apk.zip](https://github.com/fossasia/pslab-android/files/2275478/multimeter_toolbar.apk.zip)

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@woshikie @CloudyPadmal @harsh-2711 @Avjeet Please test the apks in your devices and let me know if there is any issue in the UI w.r.t screen-sizes

**Comment by abhinavraj23** [comment]
@cweitat it is the a separate toolbar made for the multimeter instrument so that we can add data-recording feature in multimeter, it is not the same toolbar used in the instrument activity. I will add the data recording feature in my subsequent PRs.

File: app/src/main/java/io/pslab/activity/MultimeterActivity.java [hunk 1]
```diff
@@ -39,6 +40,8 @@
 
     private ScienceLab scienceLab;
 
+    @BindView(R.id.accel_toolbar)
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/activity/MultimeterActivity.java]: View looks ok in my device. Why is this referring to accel_toolbar?

**Comment by abhinavraj23** [comment]
@CloudyPadmal Changes done :+1:

**Pull request merged**